assets/index.less

```diff
@@ -14,12 +14,6 @@
   color: #666;
   line-height: 28px;
 
-  &-allow-clear {
-    .@{selectPrefixCls}-selection--single .@{selectPrefixCls}-selection__rendered {
-      padding-right: 40px;
-    }
-  }
```

Comment:
It is useful to avoid text overclap of arrow and selected content.

Reply:
@afc163 I didn't get you here. Can you please explain what was the fix for the issue.

Reply:
https://github.com/ant-design/ant-design/commit/4535e9d4f15926887c4999a60b5f9d14eb701ddd

Reply:
@afc163 Its great that you fixed it but It has been 2 days since I gave the PR so I was under the impression that if a person is working on an issue and hasn't followed up for more than two weeks then some one can take it over leaving a comment as per https://ant.design/docs/react/contributing so that efforts are not duplicated. It would have been great if you could have reviewed my PR and requested the changes instead of closing it. Please correct me If I am wrong.

Reply:
Sorry, I passingly fix it in antd, you can still continue your PR in rc-select refer to ant-design/ant-design@4535e9d

Reply:
thanks @afc163. As the issue is already fixed in antd so what change are you referring to in the above comment ?